.github/workflows/build.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
```

Comment:
shoulb remove main probably if we dont need that reference, same goes for the other file